go.mod

```diff
@@ -345,6 +344,7 @@ require (
 	github.com/samber/lo v1.37.0 // indirect
 	github.com/saracen/walker v0.1.3 // indirect
 	github.com/sassoftware/go-rpmutils v0.2.0 // indirect
+	github.com/satori/go.uuid v1.2.0 // indirect
```

Comment:
So we still get the import indirect :)

Reply:
Yeah, I'd no time to track it down. I'll try to do it today to figure out if we can get rid of it completely.

Reply:
@FrimIdan It is a transient dependency for `kubeclarity`. So we need to fix this there: